src/com/softwaret/kpdf/repository/validation/input/name/NameValidationRepositoryImpl.kt

```diff
@@ -1,10 +1,13 @@
 package com.softwaret.kpdf.repository.validation.input.name
 
+import com.softwaret.kpdf.model.inline.Name
+
 class NameValidationRepositoryImpl : NameValidationRepository {
 
-    override fun validateName(name: String?) = if (name.isNullOrBlank()) {
-        NameValidationError.Generic
-    } else {
-        null
-    }
+    override fun validateName(name: Name?) =
+        if (name?.value.isNullOrBlank()) {
```

Comment:
Postaram się tu zaproponować rozwiązanie, bo te sprawdzanie czy `null` i zwracanie `null` ciągle mi się nie podoba

Reply:
Nullowalność jest tutaj dzięki lokacjom tylko dlatego że nie chciało mi się jej wywalić xD Już usuwam

Reply:
Ale z samym zwracaniem nulla/nienulla to możemy pogadać jutro

Reply:
oki doki